ceph/rados/utils.py

```diff
@@ -0,0 +1,118 @@
+"""
+   This module contains the wrapper functions to perform general ceph cluster modification operations.
+    1. Remove OSD
+    2. Add OSD
+    3. Set osd out
+    3. Zap device path
+"""
+import logging
+
+from ceph.ceph_admin.daemon import Daemon
+from ceph.ceph_admin.device import Device
+from ceph.ceph_admin.osd import OSD
+
+log = logging.getLogger(__name__)
+
+
+def method_should_succeed(function, *args, **kwargs):
+    """
+    Checks return status of command execution
+    and raise assertion on failure
+    Args:
+        function: name of the function
+        args: arg list
+        kwargs: arg dict
+    """
+    if function(*args, **kwargs):
+        pass
+    else:
+        raise AssertionError(f"Execution failed at function {function}")
```

Comment:
Can we move this method to `utils/utility.py`? This is a good addition

Reply:
will do it in next PR.